doc/source/releases/17.01.rst

```diff
@@ -174,19 +135,11 @@ Fixes
 * Dependency resolver documentation fixes
   (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
   `Pull Request 3490`_
-* Attempt to fix broken integration tests on Jenkins.
-  `Pull Request 3365`_
-* Fix auto creation of XUnit reports for Jenkins.
-  `Pull Request 3368`_
-* Fix automatic handling of Jenkins BUILD_NUMBER in Dockerized tests.
-  `Pull Request 3370`_
 * Fix dependency display and api
   (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
   `Pull Request 3427`_
```

Comment:
Sorry - I was working on this as you were commenting. I merged this in with the generic fixes just now I think.